common/include/pcl/common/impl/common.hpp

```diff
@@ -70,6 +70,20 @@ pcl::getAngle3D (const Eigen::Vector3f &v1, const Eigen::Vector3f &v2, const boo
 inline void
 pcl::getMeanStd (const std::vector<float> &values, double &mean, double &stddev)
 {
+    try
+    {
+	    // throw a exception when there are less than two elements in vector `values` 
+	    // to prevent division by zero error while calculating standard deviation.
+	    if(values.size() < 2 )
```

Comment:
Why not less than 1?

Reply:
Because standard deviation needs division by `n` and `n-1` as done in this function's last few lines.

Reply:
Yes. But the code does not handle it. I just tested and `stddev` is `nan` when a single number is passed. I wonder how did that skip testing. I'll update the code to handle it.